src/resmom/mom_main.c

```diff
@@ -474,7 +474,7 @@ static handler_ret_t	set_restart_transmogrify(char *);
 static handler_ret_t	set_restrict_user(char *);
 static handler_ret_t	set_restrict_user_maxsys(char *);
 static handler_ret_t	set_restrict_user_exceptions(char *);
-static handler_ret_t	set_secondary_pbs_nodefile(char *);
+static handler_ret_t	set_gen_nodefile_on_sister_mom(char *);
```

Comment:
per our updated coding standards, variables and declarations need not be aligned - so you need to remove the alignment of all the blocks u touch